rules-tests/Php80/Rector/Switch_/ChangeSwitchToMatchRector/Fixture/skip_unrelated_return.php.inc

```diff
@@ -20,23 +20,3 @@ final class UnrelatedReturn
 }
 
 ?>
------
-<?php
-
-namespace Rector\Tests\Php80\Rector\Switch_\ChangeSwitchToMatchRector\Fixture;
```

Comment:
this test is named `skip_unrelated_return.php.inc` but implementation wise expected a code transformation. since it is running on `switch(mixed)` I made it obey to its own name